### PR TITLE
Idlayerth

### PR DESCRIPTION
**Pull request by LonelyCat124**
v 0.0
It can detect smokes at current, but the icons are not shown in the correct place. Few other things depend on user paths so need to work them out. Hopefully i can fix those this weekend before IDL final and jack can use it but will see.

The core is very much just a "script" and to add more features i should refactor stuff but just wanted something to work :D

Uses PyQT5 so may not work on certain Linux distros

### CODE REVIEW

**Comment by LonelyCat124** [comment]
v0.1 can be merged I think

**Comment by LonelyCat124** [comment]
Works only with display capture

**Comment by alex9smith** [comment]
This is really good! I saw you took me off review - want me to hold off merging for now?

**Comment by LonelyCat124** [comment]
No i took you off assigning :P. Nah can be merged to now m ostly so i can do more pull requests

File: idlayerth/load_config.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# coding: utf-8
+import yaml
+import os
+with open("C:\\Users\\aidan\\ascendant\\idlayerth\\config.yml", 'r') as stream:
```
Comment (alex9smith) [inline idlayerth/load_config.py]: You'll need to change this bit for other people, but that's no big deal

**Pull request merged**